### DISCUSSION TITLE
thumbnail generation

### DISCUSSION THREAD

**Discussion by Sblop** [jonaswinkler/paperless-ng#547] [Support] [answered]
Hi All,

Is it possible to force thumbnail generation, like it is possible to force the document_archive function? I miss a few thumbnails after a recent data mishap, and the sanity checker informs me of this...

/Benjamin

**Comment by jonaswinkler** [comment, marked as answer]
The management command `document_thumbnails` does that. I just noticed that that's missing in the documentation.

Either call it without arguments, and it will redo *all* thumbnails, or call it with arguments (`document_thumbnails -d <id>`), where id is a document ID with missing thumbnail and do it one by one.

There's currently no logic here to just do it for missing thumbnails, sorry.

**Reply by kohfuchs** [reply]
Hi @jonaswinkler ,
is this still working?
When I run `docker-compose run --rm webserver document_thumbnails`
The following error occours:

```
Creating paperless-ng_webserver_run ... done
Waiting for PostgreSQL to start...
Operations to perform:
  Apply all migrations: admin, auth, authtoken, contenttypes, django_q, documents, paperless_mail, sessions
Running migrations:
  No migrations to apply.
Unknown command: 'document_thumbnails'. Did you mean document_logs?
Type 'manage.py help' for usage.
ERROR: 1
```

**Reply by jonaswinkler** [reply]
Are you sure you're on the latest version? `document_logs` is not supposed to exist anymore.

**Reply by kohfuchs** [reply]
You are right, i was running an old version. Now docker-compose file updated. Thank you!
